src/erlcloud_ddb2.erl

```diff
@@ -640,7 +640,9 @@ undynamize_value({<<"M">>, Map}, Opts) ->
 
 -spec undynamize_attr(json_attr(), undynamize_opts()) -> out_attr().
 undynamize_attr({Name, [ValueJson]}, Opts) ->
-    {Name, undynamize_value(ValueJson, Opts)}.
+    {Name, undynamize_value(ValueJson, Opts)};
+undynamize_attr({}, _) ->
+    {}.
```

Comment:
_Oops, Seems I accidently deleted my comment..._

`jsx` returns `[{}]` for empty objects, it may be a better way to translating `[{}]` into `[]`, like we did in:
```erlang
undynamize_object(_, [{}], _) ->
    %% jsx returns [{}] for empty objects
    [];
undynamize_object(PairFun, List, Opts) ->
    [PairFun(I, Opts) || I <- List].
```

Additionally, we need `[]` as a result of empty map, instead of `[{}]` in current fix.